byzcoin/proto.go

```diff
@@ -113,6 +113,26 @@ type GetProofResponse struct {
 	Proof Proof
 }
 
+// CheckAuthorization is used to get the list of actions authorized by a list of
+// identities.
```

Comment:
I think the comment can be improved. Maybe say "return the list of actions that could be executed if the signatures of the given identities are present and valid".